packages/@aws-cdk/aws-apigateway/README.md

```diff
@@ -546,6 +546,23 @@ const api = new apigateway.RestApi(this, 'books', {
 })
 ```
 
+You can also use the `deployOptions` property to configure
+[default method throttling](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-request-throttling.html#apigateway-api-level-throttling-in-usage-plan)
+for the deployment stage.
+
```

Comment:
I would rephrase this like below - 

You can use the `methodOptions` property on the `Stage` construct to configure [default method throttling](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-request-throttling.html#apigateway-api-level-throttling-in-usage-plan) for the deployment stage. The following snippet configures a stage whose default throttling limits are set to 100 TPS -

<insert code snippet configuring this on a `Stage`>

Configuring this on the `deployOptions` of `RestApi` will set this on the default stage that is created, like so -

<insert your current code snippet>

Reply:
Thanks for this feedback. I've made this change!